Detectors/PHOS/simulation/src/RawWriter.cxx

```diff
@@ -23,12 +22,12 @@ using namespace o2::phos;
 void RawWriter::init()
 {
   mRawWriter = std::make_unique<o2::raw::RawFileWriter>(o2::header::gDataOriginPHS, false);
-  mRawWriter->setCarryOverCallBack(this);
-  mRawWriter->setApplyCarryOverToLastPage(true);
+  //  mRawWriter->setCarryOverCallBack(this);
+  mRawWriter->setApplyCarryOverToLastPage(false);
```

Comment:
if you don't use carryOver callback, then no need to indicate ApplyCarryOverToLastPage policy either

Reply:
Commented out